### ISSUE TITLE
NullReferenceException when showing TaskDialog without buttons

### ISSUE THREAD

**Issue by mgpreston** [ookii-dialogs/ookii-dialogs-wpf#48]
If you attempt to call ShowDialog on a TaskDialog that you haven't added any buttons to, the call to ShowDialog will throw a NullReferenceException.

Could this be improved to throw a more meaningful exception - or perhaps add a default button to it?

Example:
```
var dialog = new Ookii.Dialogs.Wpf.TaskDialog {
	Content = "Message",
	MainInstruction = "Instruction"
};

dialog.ShowDialog(); // NullReferenceException here. dialog is not null.
```

Thank you!

**Comment by augustoproiete** [comment]
Thanks for reporting @mgpreston. I [fixed the `NullReferenceException`](https://github.com/ookii-dialogs/ookii-dialogs-wpf/commit/5582dcbb591a206ce893fb75d9060060370901ad) and that a more meaningful exception is thrown, but showing a default button might be a better experience.

What do you think the button should be? `OK`? `Close`? `Cancel`? Something else?

**Comment by mgpreston** [comment]
That's great, thanks!

I would be tempted to go for `OK` as the default button, but I'd consider any of them to be fine.

**Comment by augustoproiete** [comment]
Tracking adding the default button via https://github.com/ookii-dialogs/ookii-dialogs-wpf/issues/55

Closing this one as fixed by [`5582dcb`](https://github.com/ookii-dialogs/ookii-dialogs-wpf/commit/5582dcbb591a206ce893fb75d9060060370901ad)

**Comment by augustoproiete** [comment]
:tada: This issue has been resolved in **Ookii.Dialogs.Wpf v4.0.0**

The release is available on:

- :rocket: [GitHub Release](https://github.com/ookii-dialogs/ookii-dialogs-wpf/releases/tag/v4.0.0)
- :package: [NuGet Package](https://www.nuget.org/packages/Ookii.Dialogs.Wpf/4.0.0)
